code/src/index.css

```diff
@@ -1,13 +1,185 @@
+*{
+  margin: 0px;
+  padding: 0px;
+  box-sizing: border-box;
+}
+
 body {
-  margin: 0;
   font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
     "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-color: #191414;
 }
 
+
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+h1 {
+  color: white;
+  font-size: 28px;
+  padding-left: 4%;
+  padding-bottom: 30px;
+  padding-top: 30px;
+}
+
+h2 {
+  color: white;
+  font-size: 14px;
+  font-weight: bold;
+  margin: 5px 0px;
+  max-width: 300px;
+}
+
+h3{
+  color: darkgray;
+  font-size: 14px;
+
+}
+
+.albums-container {
+  display: grid;
+  align-items: end;
+  height: max-content;
+  justify-items: center;
+  align-items: stretch;
+
+}
+
+.image-container {
+  display: flex;
+  position: relative;
+  height: max-content;
+  width: 100%;
+  justify-content: center;
+}
+
+.image-container:hover .album-image {
+ opacity: 0.3;
+}
+
+.image-container:hover .icons-container {
+  opacity: 1;
+ }
+
+.album-image {
+  opacity:1;
+  display: block;
+  margin-bottom: 10px;
+}
+
+.album-card {
+display: grid;
+gap: 10px;
```

Comment:
What is the purpose of having a grid here? Or is it leftovers from your struggle with the grid.. 😉